idd/Energy+.idd.in

```diff
@@ -64868,37 +65045,257 @@ WaterHeater:HeatPump,
+  N2,  \field Condenser Bottom Location
+       \type real
+       \units m
+       \minimum 0
+       \default 0
+       \note Distance from the bottom of the tank to the bottom of the wrapped condenser.
+  N3,  \field Condenser Top Location
+       \required-field
+       \type real
+       \units m
+       \minimum 0
+       \note Distance from the bottom of the tank to the top of the wrapped condenser.
+  N4 , \field Evaporator Air Flow Rate
+       \type real
+       \units m3/s
+       \minimum> 0
+       \autocalculatable
+       \note Actual air flow rate across the heat pump's air coil (evaporator).
+       \note If autocalculated, the air flow rate is set equal to 5.035E-5 m3/s/W times
+       \note the rated heating capacity of the heat pump's DX coil.
+  A4 , \field Inlet Air Configuration
+       \required-field
+       \type choice
+       \key Schedule
+       \key ZoneAirOnly
+       \key OutdoorAirOnly
+       \key ZoneAndOutdoorAir
+       \note Defines the configuration of the airflow path through the air coil and fan section.
+  A5 , \field Air Inlet Node Name
+       \type node
+       \note Zone air exhaust node name if Inlet Air Configuration is ZoneAirOnly or
+       \note ZoneAndOutdoorAir.
+       \note Simply a unique Node Name if Inlet Air Configuration is Schedule.
+       \note Otherwise, leave field blank.
+  A6 , \field Air Outlet Node Name
+       \type node
+       \note Zone Air Inlet Node Name if Inlet Air Configuration is ZoneAirOnly or
+       \note ZoneAndOutdoorAir.
+       \note Simply a unique Node Name if Inlet Air Configuration is Schedule.
+       \note Otherwise, leave field blank.
+  A7 , \field Outdoor Air Node Name
+       \type node
+       \note Outdoor air node name if inlet air configuration is ZoneAndOutdoorAir
+       \note or OutdoorAirOnly, otherwise leave field blank.
+  A8 , \field Exhaust Air Node Name
+       \type node
+       \note Simply a unique Node Name if Inlet Air Configuration is ZoneAndOutdoorAir
+       \note or OutdoorAirOnly, otherwise leave field blank.
+  A9 , \field Inlet Air Temperature Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note Used only if Inlet Air Configuration is Schedule, otherwise leave blank.
+       \note Schedule values should be degrees C.
+  A10, \field Inlet Air Humidity Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note Used only if Inlet Air Configuration is Schedule, otherwise leave blank.
+       \note Schedule values are entered as a fraction (e.g. 0.5 is equal to 50%RH)
+  A11, \field Inlet Air Zone Name
+       \type object-list
+       \object-list ZoneNames
+       \note Used only if Inlet Air Configuration is ZoneAirOnly or ZoneAndOutdoorAir.
+       \note Otherwise, leave field blank.
+  A12, \field Tank Name
+       \required-field
+       \type object-list
+       \object-list WaterHeaterNames
```

Comment:
@nmerket If this is limited to only stratified water heaters, then a new reference is needed that only includes those.